app/(auth)/signup/_components/signupForm.tsx

```diff
@@ -0,0 +1,109 @@
+'use client';
+
+import { Controller, FieldValues, useForm } from 'react-hook-form';
+
+import { AxiosError } from 'axios';
+import { useRouter } from 'next/navigation';
+
+import Button from '@/components/button/button';
+import Input from '@/components/input/input';
+import { postSignUpInfo } from '@/services/api';
+
+import MemberType from './memberType';
+
+const EMAIL_REGEX = /^[A-Za-z0-9]([-_.]?[A-Za-z0-9])*@[a-z0-9]([-_.]?[A-Za-z0-9])*\.[A-Za-z]{2,3}$/;
+const PASSWORD_REGEX = /^.{8,}$/;
```

Comment:
관리하기 나름이지만 2개 이상이라면 이렇게 관리해줘도 괜찮을 것 같아요!
```suggestion
const REGEX = {
  EMAIL: /^[A-Za-z0-9]([-_.]?[A-Za-z0-9])*@[a-z0-9]([-_.]?[A-Za-z0-9])*\.[A-Za-z]{2,3}$/,
  PASSWORD: /^.{8,}$/,
};
```